docs/advanced/fragment-introspection.md

```diff
@@ -0,0 +1,99 @@
+# GraphQL Fragment Introspection
```

Comment:
This doc should be in the guides of the builders.

Reply:
moved